packages/genui/a2ui-playground/src/pages/DemosPage.css

```
@@ -623,6 +642,370 @@ html[data-theme="dark"] .detailBackButton:hover {
+  line-height: 1.4;
+}
+
+.playbackChunkJson {
+  margin: 0;
+  padding: 8px 10px;
+  font-family: var(--geist-mono);
+  font-size: 11px;
+  line-height: 1.55;
+  color: var(--geist-secondary);
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)